bitnami/elasticsearch/templates/data-statefulset.yaml

```diff
@@ -107,6 +110,8 @@ spec:
             - name: ELASTICSEARCH_NODE_TYPE
               value: "data"
           ports:
+            - name: http
+              containerPort: 9200
```

Comment:
Why exposing this port on data nodes? These nodes shouldn't be accessed on this port by anyone.

Reply:
Actually this port is already used by readiness and liveness [probes](https://github.com/irizzant/charts-1/blob/0d29428887f122512627ca744fce8a726c9f2254/bitnami/elasticsearch/templates/data-statefulset.yaml#L126), so there is traffic on it from other Pods (kubelet in this case).
Moreover having this port open gives the possibility to use REST API with these nodes too whenever needed.

Reply:
Okay with adding the port for the container since it's used for probes